Makefile.toml

```diff
@@ -2,6 +2,7 @@
 [env]
 CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = true
 CARGO_TARGET_DIR = "${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY}/target"
+CARGO_MAKE_WORKSPACE_SKIP_MEMBERS = ["zellij-utils", "zellij-server", "zellij-tile", "zellij-client", "zellij-tile-utils"]
```

Comment:
I'm not sure how I feel about this exactly, I've been using the `workspace = false` on tasks that should only be done once. I'm a touch worried this will break things like running clippy on all crates.

Additionally, I'm not a massive fan of having a `test` feature, but I also don't know if there is a better way. Do the other crates need to expose different functionality for `zellij` to have tests?

Reply:
Overall though, I like this reorganization! As long as Clippy, running all tests, and the release building still works, I'm happy :)

Reply:
> I'm not sure how I feel about this exactly, I've been using the `workspace = false` on tasks that should only be done once. I'm a touch worried this will break things like running clippy on all crates.

Yeah, that's reasonable. I'll revert this change. The compile-time also hasn't reduced much.
 
> Additionally, I'm not a massive fan of having a `test` feature, but I also don't know if there is a better way. Do the other crates need to expose different functionality for `zellij` to have tests?

Yeah, we have used `#[cfg(test)]` and `#[cfg(not(test))]` in a number of places which are now in the 3 different crates. But I think things will improve as the multiprocess model develops and a new testing framework is adopted and hopefully we won't need the `test` feature in the future.

Reply:
Although, my one tiny concern is that cargo-make build everything with `--all-features` flag. So when you run `cargo make run/build`, it will first build the 3 `zellij-*` crates with the `test` feature enabled, and later on it builds them as a dependency of `zellij` without the `test` feature. Would it be alright to remove the `--all-features` flag? (Not a big deal though)

Reply:
Yeah, I'm happy to remove the all-features flag! Thanks for testing all of this!